omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java

```diff
@@ -64,6 +67,8 @@
 	public static final String GLOBAL_PROPERTY_USER = "dhisconnector.user";
 
 	public static final String GLOBAL_PROPERTY_PASS = "dhisconnector.pass";
+	
+	public static final String GLOBAL_PROPERTY_FULL_MAPPING  = "dhisconnector.fullmappingvalues";
```

Comment:
why not initialise this in config.xml, better you even have it in one place and reference it from there where else you need it

Reply:
I wanted to be coherent with the other codes (user and password preoperties)

Reply:
those (user & pass) are initialised in config.xml